Esignet/scripts/Esignet_Test_Script.jmx

```diff
@@ -1678,9 +1883,62 @@ vars.put(&quot;utcTime&quot;, utcTime);</stringProp>
             <intProp name="Assertion.test_type">2</intProp>
           </ResponseAssertion>
           <hashTree/>
+          <BeanShellPreProcessor guiclass="TestBeanGUI" testclass="BeanShellPreProcessor" testname="Generate UTC Time" enabled="true">
+            <stringProp name="filename"></stringProp>
+            <stringProp name="parameters"></stringProp>
+            <boolProp name="resetInterpreter">false</boolProp>
+            <stringProp name="script">import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.TimeZone;//Set the format for the date-time
+SimpleDateFormat dateFormat = new SimpleDateFormat(&quot;yyyy-MM-dd&apos;T&apos;HH:mm:ss.sss&quot;);
+dateFormat.setTimeZone(TimeZone.getTimeZone(&quot;UTC&quot;));// Get the current time in UTC
+Calendar cal = Calendar.getInstance();
+String utcTime = dateFormat.format(cal.getTime());
+vars.put(&quot;utcTime&quot;, utcTime);
+
+</stringProp>
+          </BeanShellPreProcessor>
+          <hashTree/>
+          <BeanShellPostProcessor guiclass="TestBeanGUI" testclass="BeanShellPostProcessor" testname="Store TransactionID To File" enabled="true">
+            <stringProp name="filename"></stringProp>
+            <stringProp name="parameters"></stringProp>
+            <boolProp name="resetInterpreter">false</boolProp>
+            <stringProp name="script">${__StringToFile(transaction_id_auth_code_password_auth.txt,${tidAuthCodePrep}\,${hashValue}\n,true,)}</stringProp>
+          </BeanShellPostProcessor>
+          <hashTree/>
+          <JSR223PostProcessor guiclass="TestBeanGUI" testclass="JSR223PostProcessor" testname="Store Successful Credentials To File" enabled="true">
+            <stringProp name="scriptLanguage">groovy</stringProp>
+            <stringProp name="parameters"></stringProp>
+            <stringProp name="filename"></stringProp>
+            <stringProp name="cacheKey">true</stringProp>
+            <stringProp name="script">String expectedResponseMessage = &quot;errors\&quot;:[]&quot;;
+
+
+if (prev.getResponseDataAsString().contains(expectedResponseMessage)) {
+    
+    String identifier = vars.get(&quot;individualIdPwdAuth&quot;);
+    String password = vars.get(&quot;passwordPwdAuth&quot;);
+
+   
+    String csvFilePath = &quot;C:/Users/61091460/Documents/apache-jmeter-5.3/apache-jmeter-5.3/bin/credentials.txt&quot;;
```

Comment:
It should be relatiave path, we cannot hardcode a obsolute path since it might need a change if run in linux machine. Create a Jira task to take care of this in the next PR and add this point in to the develop guid book